testsuite/MDAnalysisTests/analysis/test_rms.py

```diff
@@ -73,6 +73,14 @@ def test_superposition(self):
         rmsd = rms.rmsd(a, b, superposition=True)
         assert_almost_equal(rmsd, 6.820321761927005)
 
+    def test_weights(self):
```

Comment:
please add a tests that you get the same results using weights with and without `superposition=True`. Then you are comparing the both RMSD algorithms to each other.

I have shown in the tests above how this is done.